src/client/eppo-client.ts

```diff
@@ -438,7 +438,8 @@ export default class EppoClient {
     subjectAttributes: BanditSubjectAttributes,
     actions: BanditActions,
     defaultValue: string,
-  ): { variation: string; action: string | null } {
+  ): { variation: string; action: string | null; evaluationDetails: IFlagEvaluationDetails } {
```

Comment:
I don't love how this doesn't parallel the other methods. I think we should leave this method & signature as-is and then make a `getBanditActionDetails()` that returns the same details object everything else does (e.g., everything--getAssignment methods and bandit methods returns a `IFlagEvaluationDetails`).

Reply:
I don't love this either. I'll update.

Reply:
There's not a lot of great options here. 

We could add an optional `banditAction` property that you suggest:
```
export interface IAssignmentDetails<T extends Variation['value'] | object>
  extends IFlagEvaluationDetails {
  value: T;
  banditAction?: string;
}

  public getBanditActionDetails(
    flagKey: string,
    subjectKey: string,
    subjectAttributes: BanditSubjectAttributes,
    actions: BanditActions,
    defaultValue: string,
  ): IAssignmentDetails<string> {
...
```
But the `value` and `banditAction` naming isn't consistent with the `getBanditActions` return type (`{ variation: string; action: string | null }`)
```

We could rename `value` to `variation`, but there's already a `variationValue` in the `IFlagEvaluationDetails`, so that would be confusing.

We could treat `{ variation: string; action: string | null }` as an entire value so that we'd have this:

```
  public getBanditAction(
    flagKey: string,
    subjectKey: string,
    subjectAttributes: BanditSubjectAttributes,
    actions: BanditActions,
    defaultValue: string,
  ): { variation: string; action: string | null } {
    return this.getBanditActionDetails(
      flagKey,
      subjectKey,
      subjectAttributes,
      actions,
      defaultValue,
    ).value;
  }

  public getBanditActionDetails(
    flagKey: string,
    subjectKey: string,
    subjectAttributes: BanditSubjectAttributes,
    actions: BanditActions,
    defaultValue: string,
  ): IAssignmentDetails<{ variation: string; action: string | null }> {
```
but then then "value" wouldn't be a consistent type with the supplied `defaultValue`.

I think the best course of action for consistency and to reduce any confusion is to make all other assignment functions have an `evaluationDetails` key. I'll update as a separate commit so that we can revert if it doesn't look right.